libp2p/P2PMessage.h

```diff
@@ -106,5 +106,22 @@ enum AMOPPacketType
     RequestTopics = 2,
     SendTopics = 3
 };
+enum TopicStatus
+{
+    INIT_STATUS = 0,
+    ENABLE_STATUS = 1,
+    DISABLE_STATUS = 2,
+};
+
+class TopicItem
```

Comment:
what about put `class TopicItem` in libdevcore, otherwise you will make libchannelserver depend on libp2p which is not reasonable.

Reply:
done